src/test/resources/ide/language/latex/Example.tex

```diff
@@ -4,11 +4,42 @@
 
 % It are not working in comnts
 
-%\begin{document} FIXME begin document crash test environment
+\begin{frame} % FIXME \begin{document} crash test environment
     \begin{itemize}
         \item $It are not checked in <warning descr="MORFOLOGIK_RULE_EN_US">formuls</warning>$
         \item It is <warning descr="ARTICLE_MISSING">friend</warning> of human.
         \item It is <warning descr="MORFOLOGIK_RULE_EN_US">frend</warning> of human
         \item It <warning descr="IT_VBZ">are</warning> working for <warning descr="MUCH_COUNTABLE">much</warning> warnings
     \end{itemize}
-%\end{document}
+
+    Our goal was to minimize the code length, it <warning descr="IT_VBZ">are</warning> natural to compare the result with other methods.
+    We have used arithmetic coding (AC) as (categories) <warning descr="EN_A_VS_AN">a</warning> entropy encoder.
+    <warning descr="UNLIKELY_OPENING_PUNCTUATION">,</warning> bad start of the sentence.
+
+    \begin{itemize}
+        \item are not bad
+        \item two
+        \item three
+    \end{itemize}
```

Comment:
I would add here one more paragraph of text :)

Reply:
Added